thrift/src/test/java/com/linecorp/armeria/server/thrift/ThriftSerializationFormatsTest.java

```diff
@@ -122,8 +122,8 @@ public void notDefault() throws Exception {
     public void notAllowed() throws Exception {
         final HelloService.Iface client =
                 Clients.newClient(server.uri(TEXT, "/hellobinaryonly"), HelloService.Iface.class);
-        thrown.expect(InvalidResponseException.class);
-        thrown.expectMessage("415 Unsupported Media Type");
+        thrown.expect(InvalidResponseHeadersException.class);
+        thrown.expectMessage(":status=415");
```

Comment:
is it needed to add unit test case for server error?
like `status is (503) `?

Reply:
Sure. Let me add one.

Reply:
Added a test case to `ThriftOverHttpClientTest`

Reply:
Thanks for the unit test! LGTM